_data/simple-icons.json

```
@@ -3767,6 +3767,11 @@
             "hex": "FF7139",
             "source": "https://mozilla.design/firefox/logos-usage/"
         },
+        {
+            "title": "Fireship",
+            "hex": "FA8142",
```

I think this color is not necessarily the brand color, even if it's used for a button on the page. The page uses several colors. I'd say the brand color should be one of the two gradient stop colors in the logo, perhaps the lighter one:

```suggestion
            "hex": "EB844E",
```

WDYT?